Source/Misc/HauntSystem.cs

```diff
@@ -46,7 +46,7 @@ public static void Init()
                 }
                 // Host to target
                 else if(chats.FirstOrDefault(x => x.host.Id == args.Channel.Id) != null) {
-                    if(args.Message.Content.StartsWith("."))
+                    if(args.Message.Content.StartsWith($"{Bot.config.prefix"))
```

Comment:
"{Bot.config.prefix" 

You need a closing }... or just replace the string entirely with just "Bot.config.prefix"

Reply:
I'm a fucking donut